dubbo-rpc/dubbo-rpc-native-thrift/src/main/java/org/apache/dubbo/rpc/protocol/nativethrift/ThriftProtocol.java

```diff
@@ -129,8 +126,8 @@ public ThriftProtocol(Class<?>... exceptions) {
             logger.error("Fail to create nativethrift server(" + url + ") due to null args");
             throw new RpcException("Fail to create nativethrift server(" + url + ") due to null args");
         }
-        final TServer thriftServer =  new TThreadedSelectorServer(tArgs);
-        serverMap.put(url.getAddress(),thriftServer);
+        final TServer thriftServer = new TThreadedSelectorServer(tArgs);
+        serverMap.put(url.getAddress(), thriftServer);
```

Comment:
Do we have a method(such as a pattern?) to make all those formatter done? I found many this kind of problems.
Like https://github.com/apache/dubbo/blob/87703adb16a354456e89724b2ec6dd9e3d02de9c/dubbo-plugin/dubbo-qos/src/main/java/org/apache/dubbo/qos/command/CommandContextFactory.java#L25

Reply:
Different protocols have different implementations, and I don't think there is a need to use design patterns here.

Reply:
I mean the lack of space here, not design pattern, but code format.

Reply:
Sorry, understand the mistake, modify it later.